test/functional/inspec_exec_json_test.rb

```diff
@@ -11,7 +11,7 @@
     data = JSON.parse(out.stdout)
     sout = inspec('schema exec-json')
     schema = JSON.parse(sout.stdout)
-    JSON::Schema.validate(data, schema)
+    JSON::Validator.validate(schema, data).wont_equal false
```

Comment:
Yes! Thank you for checking the result.

Reply:
Right, if we aren't going to check the result it almost doesn't make sense to validate it at all; unless we're expecting the test to blow up?